src/triage/cli.py

```diff
@@ -120,6 +122,10 @@ def __init__(self, parser):
             type=argparse.FileType('r'),
             help="config file for Experiment"
         )
+        parser.add_argument(
+            '-s', '--setup',
+            help="python module to import before running the Experiment",
```

Comment:
If you want this to be specified at the root level, (currently along with `dbfile`), it needs to be in `Triage.__init__`. (And that would go along with discussion below.)

Reply:
Thank you for all your suggestions!